### PR TITLE
[Perl5] Started

### PR DESCRIPTION
**Pull request by rfilipo**
The maintainers are talking and committed to the new v3 effort.

### CODE REVIEW

File: languages/perl5/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This area will contain everything needed to launch the Perl 5 track, including:
 
 Before we publicize requesting contribution for this language, the following steps should be done.
 
-- [ ] Have a kick-off discussion between track maintainers
+- [x] Have a kick-off discussion between track maintainers
 - [ ] Fill out the [maintainers.md](./maintainers.md) file (e.g. [C#](../csharp/maintainers.md))
```
Comment (kotp) [inline languages/perl5/README.md]: Should this link be fixed to reflect the actual file needed to edit?

**Review by kotp — DISMISSED** [review]

File: languages/perl5/README.md (hunk 1 above)
Comment (rfilipo) [reply languages/perl5/README.md]: What link, @kotp ? Sorry, I didn't understand. Do you say the _maintainers.md_ file?

Comment (kotp) [reply languages/perl5/README.md]: `[C#](../csharp/maintainers.md)` is the linked file... should we link it to the actual file that should be filled out, per the check box?

**Review by ErikSchierboom — APPROVED** [review]
LGTM

File: languages/perl5/README.md (hunk 1 above)
Comment (ErikSchierboom) [reply languages/perl5/README.md:16]: This should be fixed, possibly in a separate PR.

**Comment by ErikSchierboom** [comment]
This Friday (Jan 29th 2021), we're going to explode this repo back into the individual track repositories and archive this repo.

As such, this PR needs to be merged by Friday 9am UTC, or it will need to be manually reopened against the track repo.

**Pull request closed without merging**